.github/workflows/release.yaml

```diff
@@ -0,0 +1,59 @@
+name: release
+
+on:
+  workflow_dispatch:
```

Comment:
When releasing with `workflow_dispatch` I normally make it customizable:
- the version number to be released
- the branch to release from

e.g.: https://github.com/kiota-community/kiota-java-extra/blob/8b9f209e22ecf3490d0d4e18bad47bf4695bb52e/.github/workflows/release.yml#L6-L12